crypt-common.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright (C) 2018 Björn Esser <besser82@fedoraproject.org>
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE AUTHOR AND CONTRIBUTORS ``AS IS'' AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED.  IN NO EVENT SHALL THE AUTHOR OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
+ * OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+ * HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ * LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+ * OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
+ * SUCH DAMAGE.
+ */
+
+/* Simple commonly used helper constants.  */
+
+#ifndef _CRYPT_COMMON_H
+#define _CRYPT_COMMON_H 1
+
+/* Table with characters for base64 transformation.  */
+const unsigned char ascii64[65];
+
+/* Same table gets used with other names in various places.  */
+#define b64t   ascii64
+#define itoa64 ascii64
```

Comment:
You're doing this to minimize the number of changes we make to the `.c` files that are imported from elsewhere, right?  It looks like all of the places that use the name `b64t` want plain `char`, so you could eliminate all of the explicit casts that you introduced by having it be

``` c
#define b64t   ((const char *)ascii64)
```

Reply:
Likewise.